doc/release-notes-6723.md

```diff
@@ -0,0 +1,9 @@
+Updated RPCs
+------------
+
+* The RPCs `protx register_legacy`, `protx register_fund_legacy`, `protx register_prepare_legacy` and
+  `protx update_registrar_legacy` have been deprecated in Dash Core v23 and may be removed in a future version
```

Comment:
we unlikely remove this RPC if you won't find a way to re-implement feature_dip3_v19 for "pre-v19" status.

Otherwise any bug in legacy bls will be possible to find only by re-indexing old blocks which is a bit heavy process

Reply:
This RPC is most likely to be kept around for debugging and testing but users and developers should not rely on that. It being kept around does not preclude validation rules that prohibit the creation of _new_ legacy scheme masternodes.